ubuntu_install.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+
+sudo apt-get install figlet
+
+echo "Nephos" | figlet
```

Comment:
Again this is installing a dependency for nothing. Someone might be with limited bandwith or space and you are wasting those resources